external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerKrbJDBCIntegrationSuite.scala

```diff
@@ -107,7 +109,7 @@ abstract class DockerKrbJDBCIntegrationSuite extends DockerJDBCIntegrationSuite
     conn.prepareStatement("INSERT INTO bar VALUES ('hello')").executeUpdate()
   }
 
-  test("Basic read test in query option") {
```

Comment:
@sarutak, there's actually a better way with less changes by overriding `test` method. e.g.) https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandTestUtils.scala#L52-L55

Reply:
Thanks. It seems simpler.